examples/rov_whylogs/README.md

```diff
@@ -0,0 +1,13 @@
+# Fault Detection of an Underwater Drone with Kafka and whylogs
+
+This is the project's repository for the following article:
+
```

Comment:
We should just add the link here once the article's published

Reply:
+1